fixtures/conch-v3/device.yaml

```diff
@@ -1390,7 +1390,7 @@ interactions:
     url: http://10.51.54.42:5000/device/dd559c03-8901-4250-9946-0cdcf00d953c/validation_state
     method: GET
   response:
-    body: '[]'
+    body: '{"error":"Entity Not Found"}'
     headers:
```

Comment:
Content-Length is also changing; Location header is not returned with 404 responses.

Reply:
Should be fixed now

Reply:
I'm not sure how you got 41 here:

```
perl -wle'print length(q!{"error":"Entity Not Found"}!)'
28
```

Reply:
huh, highlighted it in vim and did `<,>!wc -c` ... I'll change it

Reply:
that would have sent the entire line to wc.

Reply:
turns out I needed `wc -m` not `-c`